### DISCUSSION TITLE
Massive schema file is hard to use with code control

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#16100] [Feature Requests]
Due to the size of the schema snapshot, it is hard to process conflicts and changes between multiple developers and environments.

Not only that, but as soon as one field changes the entire schema file gets touched.

Suggestion is to be able to have each collection in its own file and thereby getting better overview over what collections change etc.

**Comment by u12206050** [comment]
ENDING discussion in favour of #13041

**Comment by azrikahar** [comment]
Linking relevant discussions here: #11262, #12179, and semi-related to #9214.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
